### ISSUE TITLE
RNA-seq data in many mouse species and tissues

### ISSUE THREAD

**Issue by julien-roux** [BgeeDB/expression-annotations#122]
http://elifesciences.org/content/5/e09977v2
This seems a very nice experience to annotate (although no need to hurry, it will not be for Bgee v14)

**Comment by ANiknejad** [comment]
thank you, Julien, please see also this page
https://github.com/BgeeDB/expression-annotations/wiki/Expression-data%2C-links-to-experiment-to-annotate

**Comment by fbastian** [comment]
I think it's better to create an issue to list a new experiment to annotate, rather than writing it in the wiki. What do you think @ANiknejad?

**Comment by ANiknejad** [comment]
yes, but I like to have a page where people can put the information before WE create an issue for data annotation

**Comment by ANiknejad** [comment]
the data are here:
http://datadryad.org/resource/doi:10.5061/dryad.8jb83

I do not figure out the way to retrieve the sample information about structure and stage?
